frontend/src/index.html.ejs

```diff
@@ -63,6 +63,7 @@
 		/>
 
 		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link href='https://fonts.googleapis.com/css?family=Open Sans' rel='stylesheet'>

```

Comment:
We should be using only Inter or Space Mono

Reply:
As it was part of the old design forgot to remove it. Removed it. ✅